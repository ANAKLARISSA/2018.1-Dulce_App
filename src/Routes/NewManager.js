import React from 'react';
import NewManagerNavigator from '../Navigators/NewManagerNavigator';

class NewManager extends React.Component {

  render() {
    return (
      <NewManagerNavigator />
    );
  }
}

export default NewManager;
