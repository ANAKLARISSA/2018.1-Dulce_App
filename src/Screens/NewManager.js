import React from 'react';
import {Text, View, TextInput, Image, Alert, ScrollView} from 'react-native';
import ValidationComponent from 'react-native-form-validator';
import AGRButton from '../Components/AGRButton';


const logo = require('../../assets/img/logo.png');

const styles = {
  logo: {
    alignSelf: 'center',
    marginBottom: 20,
    width: 200,
    height: 100,
    resizeMode: 'contain'
  },
  container: {
    flex: 1,
    flexDirection: 'column',
    padding: 15,
    backgroundColor: '#FFF',
    paddingHorizontal: 50,

  },
  input: {
    height: 36,
    borderBottomWidth: 1,
    borderBottomColor: 'grey',
    marginBottom: 10,
    width: '100%',
    alignSelf: 'center',
    marginBottom: 20,
  },
  button: {
    marginTop : 30
  },
  error:{
    color: 'red',
    alignSelf: 'center'
  }
};

export default class NewManager extends ValidationComponent {

  constructor(props) {
    super(props);
    this.state = {
      nome: '',
      matricula: '',
      hospital: '',
      setor: '',
      senha: '',
      confirmaSenha: ''
    }
  }

  _onPressButton(){
    if(this.validate({
        nome: {required: true},
        matricula: {numbers: true, required: true},
        hospital: {required: true},
        setor: {required: true},
        senha: {minlength:4, maxlength:8, required: true},
        confirmaSenha: {required: true}
      }) && (this.state.confirmaSenha == this.state.senha)){
        //Metodo para mandar para a API estará aqui
        Alert.alert(
          'Dados do novo gestor',
          'nome: ' + this.state.nome +
          '\nMatricula: ' + this.state.matricula +
          '\nHospital: ' + this.state.hospital +
          '\nSetor: ' + this.state.setor +
          '\nSenha: ' + this.state.senha +
          '\nConfirmação de Senha: ' + this.state.confirmaSenha,
          [
            {text:'Ok', onPress: () => this.props.navigation.navigate('login')}
          ]
         )
      }
  }

  confirmaSenhaErrado(){
    if (this.state.confirmaSenha != this.state.senha) {
      return true;
    }else {
      return false;
    }

  }

  render() {
    return (
      <View style={styles.container}>
        <Image source={logo} style={styles.logo} />
        <ScrollView>
          <TextInput ref="nome" style={styles.input} placeholder="Nome Completo" underlineColorAndroid="transparent" onChangeText={(text) => this.setState({nome: text})} />
          {this.isFieldInError('nome') && this.getErrorsInField('nome').map(errorMessage => <Text style={styles.error}>{errorMessage}</Text>)}
          <TextInput ref="matricula" style={styles.input} placeholder="Matrícula" underlineColorAndroid="transparent" onChangeText={(text) => this.setState({matricula: text})} />
          {this.isFieldInError('matricula') && this.getErrorsInField('matricula').map(errorMessage => <Text style={styles.error}>{errorMessage}</Text>)}
          <TextInput ref="hospital" style={styles.input} placeholder="Hospital" underlineColorAndroid="transparent" onChangeText={(text) => this.setState({hospital: text})} />
          {this.isFieldInError('hospital') && this.getErrorsInField('hospital').map(errorMessage => <Text style={styles.error}>{errorMessage}</Text>)}
          <TextInput ref="setor" style={styles.input} placeholder="Setor" underlineColorAndroid="transparent" onChangeText={(text) => this.setState({setor: text})} />
          {this.isFieldInError('setor') && this.getErrorsInField('setor').map(errorMessage => <Text style={styles.error}>{errorMessage}</Text>)}
          <TextInput ref="senha" style={styles.input} placeholder="Senha" underlineColorAndroid="transparent" secureTextEntry onChangeText={(text) => this.setState({senha: text})} />
          {this.isFieldInError('senha') && this.getErrorsInField('senha').map(errorMessage => <Text style={styles.error}>{errorMessage}</Text>)}
          <TextInput ref="confirmaSenha" style={styles.input}
            placeholder="Confirme sua senha"
            underlineColorAndroid="transparent"
            secureTextEntry
            onChangeText={(text) => this.setState({confirmaSenha: text})} />
          {this.isFieldInError('confirmaSenha') && this.getErrorsInField('confirmaSenha').map(errorMessage => <Text style={styles.error}>{errorMessage}</Text>)}
          <AGRButton style={styles.button} text='Criar' onPress={() => this._onPressButton()} />
        </ScrollView>
      </View>
    );
  }
}
