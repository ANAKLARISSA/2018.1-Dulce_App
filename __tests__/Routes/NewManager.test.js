import React from 'react';
import renderer from 'react-test-renderer';
import NewManager from '../../src/Routes/NewManager';

test('renders correctly' , () => {
  const tree = renderer
  .create(<NewManager />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
