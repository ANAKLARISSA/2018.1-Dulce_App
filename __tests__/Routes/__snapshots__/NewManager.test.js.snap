// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
    ]
  }
>
  <View
    onMoveShouldSetResponder={[Function]}
    onMoveShouldSetResponderCapture={[Function]}
    onResponderEnd={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderReject={[Function]}
    onResponderRelease={[Function]}
    onResponderStart={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    onStartShouldSetResponderCapture={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
          "flexDirection": "column-reverse",
        },
        Object {
          "backgroundColor": "#000",
        },
      ]
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        collapsable={undefined}
        pointerEvents="auto"
        style={
          Object {
            "backgroundColor": "#EFEFF4",
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "shadowColor": "black",
            "shadowOffset": Object {
              "height": 0,
              "width": 0,
            },
            "shadowOpacity": 0.2,
            "shadowRadius": 5,
            "top": 0,
            "transform": Array [
              Object {
                "translateX": 0,
              },
              Object {
                "translateY": 0,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#FFF",
              "flex": 1,
              "flexDirection": "column",
              "padding": 15,
            }
          }
        >
          <View>
            <Image
              source={
                Object {
                  "testUri": "../../../assets/img/DulceHorizontalFull.png",
                }
              }
              style={
                Object {
                  "alignSelf": "center",
                  "backgroundColor": "transparent",
                  "height": 100,
                  "marginBottom": 20,
                  "resizeMode": "contain",
                  "width": 200,
                }
              }
            />
          </View>
          <RCTScrollView>
            <View>
              <View
                style={Object {}}
                virtual={undefined}
              >
                <View
                  accessibilityComponentType={undefined}
                  accessibilityLabel={undefined}
                  accessibilityTraits={undefined}
                  accessible={true}
                  collapsable={undefined}
                  hasTVPreferredFocus={undefined}
                  hitSlop={undefined}
                  isTVSelectable={true}
                  nativeID={undefined}
                  onLayout={undefined}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "transparent",
                      "borderBottomWidth": 1,
                      "borderColor": "#D9D5DC",
                      "borderLeftWidth": 0,
                      "borderRightWidth": 0,
                      "borderTopWidth": 0,
                      "flexDirection": "row",
                      "marginLeft": 15,
                      "marginTop": 15,
                      "opacity": 1,
                    }
                  }
                  testID={undefined}
                  tvParallaxProperties={undefined}
                >
                  <View
                    collapsable={undefined}
                    style={
                      Object {
                        "left": 0,
                        "opacity": 1,
                        "paddingBottom": undefined,
                        "paddingTop": undefined,
                        "position": "absolute",
                        "right": 0,
                        "top": 18,
                      }
                    }
                  >
                    <Text
                      accessible={true}
                      allowFontScaling={true}
                      ellipsizeMode="tail"
                      style={
                        Object {
                          "color": "#575757",
                          "fontSize": 17,
                          "left": 0,
                          "paddingRight": 5,
                          "top": 6,
                        }
                      }
                      virtual={undefined}
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "fontSize": 17,
                          }
                        }
                        virtual={undefined}
                      >
                        Nome Completo
                      </Text>
                    </Text>
                  </View>
                  <TextInput
                    allowFontScaling={true}
                    editable={true}
                    nameLabel="Nome Completo"
                    onBlur={[Function]}
                    onChangeText={[Function]}
                    onFocus={[Function]}
                    placeholderTextColor="#575757"
                    style={
                      Array [
                        Object {
                          "color": "#000",
                          "flex": 1,
                          "fontSize": 17,
                          "height": 50,
                          "lineHeight": 24,
                          "marginLeft": null,
                          "paddingLeft": null,
                          "paddingRight": 5,
                          "top": 10,
                        },
                        Object {
                          "alignSelf": "center",
                          "borderBottomColor": "grey",
                          "borderBottomWidth": 1,
                          "height": 40,
                          "marginBottom": 10,
                          "marginTop": 30,
                          "width": "100%",
                        },
                      ]
                    }
                    underlineColorAndroid="transparent"
                    value=""
                    virtual={undefined}
                  />
                </View>
              </View>
              <View
                style={Object {}}
                virtual={undefined}
              >
                <View
                  accessibilityComponentType={undefined}
                  accessibilityLabel={undefined}
                  accessibilityTraits={undefined}
                  accessible={true}
                  collapsable={undefined}
                  hasTVPreferredFocus={undefined}
                  hitSlop={undefined}
                  isTVSelectable={true}
                  nativeID={undefined}
                  onLayout={undefined}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "transparent",
                      "borderBottomWidth": 1,
                      "borderColor": "#D9D5DC",
                      "borderLeftWidth": 0,
                      "borderRightWidth": 0,
                      "borderTopWidth": 0,
                      "flexDirection": "row",
                      "marginLeft": 15,
                      "marginTop": 15,
                      "opacity": 1,
                    }
                  }
                  testID={undefined}
                  tvParallaxProperties={undefined}
                >
                  <View
                    collapsable={undefined}
                    style={
                      Object {
                        "left": 0,
                        "opacity": 1,
                        "paddingBottom": undefined,
                        "paddingTop": undefined,
                        "position": "absolute",
                        "right": 0,
                        "top": 18,
                      }
                    }
                  >
                    <Text
                      accessible={true}
                      allowFontScaling={true}
                      ellipsizeMode="tail"
                      style={
                        Object {
                          "color": "#575757",
                          "fontSize": 17,
                          "left": 0,
                          "paddingRight": 5,
                          "top": 6,
                        }
                      }
                      virtual={undefined}
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "fontSize": 17,
                          }
                        }
                        virtual={undefined}
                      >
                        Matrícula
                      </Text>
                    </Text>
                  </View>
                  <TextInput
                    allowFontScaling={true}
                    editable={true}
                    nameLabel="Matrícula"
                    onBlur={[Function]}
                    onChangeText={[Function]}
                    onFocus={[Function]}
                    placeholderTextColor="#575757"
                    style={
                      Array [
                        Object {
                          "color": "#000",
                          "flex": 1,
                          "fontSize": 17,
                          "height": 50,
                          "lineHeight": 24,
                          "marginLeft": null,
                          "paddingLeft": null,
                          "paddingRight": 5,
                          "top": 10,
                        },
                        Object {
                          "alignSelf": "center",
                          "borderBottomColor": "grey",
                          "borderBottomWidth": 1,
                          "height": 40,
                          "marginBottom": 10,
                          "marginTop": 30,
                          "width": "100%",
                        },
                      ]
                    }
                    underlineColorAndroid="transparent"
                    value=""
                    virtual={undefined}
                  />
                </View>
              </View>
              <View
                style={Object {}}
                virtual={undefined}
              >
                <View
                  accessibilityComponentType={undefined}
                  accessibilityLabel={undefined}
                  accessibilityTraits={undefined}
                  accessible={true}
                  collapsable={undefined}
                  hasTVPreferredFocus={undefined}
                  hitSlop={undefined}
                  isTVSelectable={true}
                  nativeID={undefined}
                  onLayout={undefined}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "transparent",
                      "borderBottomWidth": 1,
                      "borderColor": "#D9D5DC",
                      "borderLeftWidth": 0,
                      "borderRightWidth": 0,
                      "borderTopWidth": 0,
                      "flexDirection": "row",
                      "marginLeft": 15,
                      "marginTop": 15,
                      "opacity": 1,
                    }
                  }
                  testID={undefined}
                  tvParallaxProperties={undefined}
                >
                  <View
                    collapsable={undefined}
                    style={
                      Object {
                        "left": 0,
                        "opacity": 1,
                        "paddingBottom": undefined,
                        "paddingTop": undefined,
                        "position": "absolute",
                        "right": 0,
                        "top": 18,
                      }
                    }
                  >
                    <Text
                      accessible={true}
                      allowFontScaling={true}
                      ellipsizeMode="tail"
                      style={
                        Object {
                          "color": "#575757",
                          "fontSize": 17,
                          "left": 0,
                          "paddingRight": 5,
                          "top": 6,
                        }
                      }
                      virtual={undefined}
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "fontSize": 17,
                          }
                        }
                        virtual={undefined}
                      >
                        Hospital
                      </Text>
                    </Text>
                  </View>
                  <TextInput
                    allowFontScaling={true}
                    editable={true}
                    nameLabel="Hospital"
                    onBlur={[Function]}
                    onChangeText={[Function]}
                    onFocus={[Function]}
                    placeholderTextColor="#575757"
                    style={
                      Array [
                        Object {
                          "color": "#000",
                          "flex": 1,
                          "fontSize": 17,
                          "height": 50,
                          "lineHeight": 24,
                          "marginLeft": null,
                          "paddingLeft": null,
                          "paddingRight": 5,
                          "top": 10,
                        },
                        Object {
                          "alignSelf": "center",
                          "borderBottomColor": "grey",
                          "borderBottomWidth": 1,
                          "height": 40,
                          "marginBottom": 10,
                          "marginTop": 30,
                          "width": "100%",
                        },
                      ]
                    }
                    underlineColorAndroid="transparent"
                    value=""
                    virtual={undefined}
                  />
                </View>
              </View>
              <View
                style={Object {}}
                virtual={undefined}
              >
                <View
                  accessibilityComponentType={undefined}
                  accessibilityLabel={undefined}
                  accessibilityTraits={undefined}
                  accessible={true}
                  collapsable={undefined}
                  hasTVPreferredFocus={undefined}
                  hitSlop={undefined}
                  isTVSelectable={true}
                  nativeID={undefined}
                  onLayout={undefined}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "transparent",
                      "borderBottomWidth": 1,
                      "borderColor": "#D9D5DC",
                      "borderLeftWidth": 0,
                      "borderRightWidth": 0,
                      "borderTopWidth": 0,
                      "flexDirection": "row",
                      "marginLeft": 15,
                      "marginTop": 15,
                      "opacity": 1,
                    }
                  }
                  testID={undefined}
                  tvParallaxProperties={undefined}
                >
                  <View
                    collapsable={undefined}
                    style={
                      Object {
                        "left": 0,
                        "opacity": 1,
                        "paddingBottom": undefined,
                        "paddingTop": undefined,
                        "position": "absolute",
                        "right": 0,
                        "top": 18,
                      }
                    }
                  >
                    <Text
                      accessible={true}
                      allowFontScaling={true}
                      ellipsizeMode="tail"
                      style={
                        Object {
                          "color": "#575757",
                          "fontSize": 17,
                          "left": 0,
                          "paddingRight": 5,
                          "top": 6,
                        }
                      }
                      virtual={undefined}
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "fontSize": 17,
                          }
                        }
                        virtual={undefined}
                      >
                        Setor
                      </Text>
                    </Text>
                  </View>
                  <TextInput
                    allowFontScaling={true}
                    editable={true}
                    nameLabel="Setor"
                    onBlur={[Function]}
                    onChangeText={[Function]}
                    onFocus={[Function]}
                    placeholderTextColor="#575757"
                    style={
                      Array [
                        Object {
                          "color": "#000",
                          "flex": 1,
                          "fontSize": 17,
                          "height": 50,
                          "lineHeight": 24,
                          "marginLeft": null,
                          "paddingLeft": null,
                          "paddingRight": 5,
                          "top": 10,
                        },
                        Object {
                          "alignSelf": "center",
                          "borderBottomColor": "grey",
                          "borderBottomWidth": 1,
                          "height": 40,
                          "marginBottom": 10,
                          "marginTop": 30,
                          "width": "100%",
                        },
                      ]
                    }
                    underlineColorAndroid="transparent"
                    value=""
                    virtual={undefined}
                  />
                </View>
              </View>
              <View
                style={Object {}}
                virtual={undefined}
              >
                <View
                  accessibilityComponentType={undefined}
                  accessibilityLabel={undefined}
                  accessibilityTraits={undefined}
                  accessible={true}
                  collapsable={undefined}
                  hasTVPreferredFocus={undefined}
                  hitSlop={undefined}
                  isTVSelectable={true}
                  nativeID={undefined}
                  onLayout={undefined}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "transparent",
                      "borderBottomWidth": 1,
                      "borderColor": "#D9D5DC",
                      "borderLeftWidth": 0,
                      "borderRightWidth": 0,
                      "borderTopWidth": 0,
                      "flexDirection": "row",
                      "marginLeft": 15,
                      "marginTop": 15,
                      "opacity": 1,
                    }
                  }
                  testID={undefined}
                  tvParallaxProperties={undefined}
                >
                  <View
                    collapsable={undefined}
                    style={
                      Object {
                        "left": 0,
                        "opacity": 1,
                        "paddingBottom": undefined,
                        "paddingTop": undefined,
                        "position": "absolute",
                        "right": 0,
                        "top": 18,
                      }
                    }
                  >
                    <Text
                      accessible={true}
                      allowFontScaling={true}
                      ellipsizeMode="tail"
                      style={
                        Object {
                          "color": "#575757",
                          "fontSize": 17,
                          "left": 0,
                          "paddingRight": 5,
                          "top": 6,
                        }
                      }
                      virtual={undefined}
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "fontSize": 17,
                          }
                        }
                        virtual={undefined}
                      >
                        Senha
                      </Text>
                    </Text>
                  </View>
                  <TextInput
                    allowFontScaling={true}
                    editable={true}
                    nameLabel="Senha"
                    onBlur={[Function]}
                    onChangeText={[Function]}
                    onFocus={[Function]}
                    placeholderTextColor="#575757"
                    secureTextEntry={true}
                    style={
                      Array [
                        Object {
                          "color": "#000",
                          "flex": 1,
                          "fontSize": 17,
                          "height": 50,
                          "lineHeight": 24,
                          "marginLeft": null,
                          "paddingLeft": null,
                          "paddingRight": 5,
                          "top": 10,
                        },
                        Object {
                          "alignSelf": "center",
                          "borderBottomColor": "grey",
                          "borderBottomWidth": 1,
                          "height": 40,
                          "marginBottom": 10,
                          "marginTop": 30,
                          "width": "100%",
                        },
                      ]
                    }
                    underlineColorAndroid="transparent"
                    value=""
                    virtual={undefined}
                  />
                </View>
              </View>
              <View
                style={Object {}}
                virtual={undefined}
              >
                <View
                  accessibilityComponentType={undefined}
                  accessibilityLabel={undefined}
                  accessibilityTraits={undefined}
                  accessible={true}
                  collapsable={undefined}
                  hasTVPreferredFocus={undefined}
                  hitSlop={undefined}
                  isTVSelectable={true}
                  nativeID={undefined}
                  onLayout={undefined}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "transparent",
                      "borderBottomWidth": 1,
                      "borderColor": "#D9D5DC",
                      "borderLeftWidth": 0,
                      "borderRightWidth": 0,
                      "borderTopWidth": 0,
                      "flexDirection": "row",
                      "marginLeft": 15,
                      "marginTop": 15,
                      "opacity": 1,
                    }
                  }
                  testID={undefined}
                  tvParallaxProperties={undefined}
                >
                  <View
                    collapsable={undefined}
                    style={
                      Object {
                        "left": 0,
                        "opacity": 1,
                        "paddingBottom": undefined,
                        "paddingTop": undefined,
                        "position": "absolute",
                        "right": 0,
                        "top": 18,
                      }
                    }
                  >
                    <Text
                      accessible={true}
                      allowFontScaling={true}
                      ellipsizeMode="tail"
                      style={
                        Object {
                          "color": "#575757",
                          "fontSize": 17,
                          "left": 0,
                          "paddingRight": 5,
                          "top": 6,
                        }
                      }
                      virtual={undefined}
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "fontSize": 17,
                          }
                        }
                        virtual={undefined}
                      >
                        Confirme sua senha
                      </Text>
                    </Text>
                  </View>
                  <TextInput
                    allowFontScaling={true}
                    editable={true}
                    nameLabel="Confirme sua senha"
                    onBlur={[Function]}
                    onChangeText={[Function]}
                    onFocus={[Function]}
                    placeholderTextColor="#575757"
                    secureTextEntry={true}
                    style={
                      Array [
                        Object {
                          "color": "#000",
                          "flex": 1,
                          "fontSize": 17,
                          "height": 50,
                          "lineHeight": 24,
                          "marginLeft": null,
                          "paddingLeft": null,
                          "paddingRight": 5,
                          "top": 10,
                        },
                        Object {
                          "alignSelf": "center",
                          "borderBottomColor": "grey",
                          "borderBottomWidth": 1,
                          "height": 40,
                          "marginBottom": 10,
                          "marginTop": 30,
                          "width": "100%",
                        },
                      ]
                    }
                    underlineColorAndroid="transparent"
                    value=""
                    virtual={undefined}
                  />
                </View>
              </View>
              <View
                style={
                  Object {
                    "flex": 1,
                    "height": 1334,
                  }
                }
                virtual={undefined}
              >
                <RCTScrollView
                  automaticallyAdjustContentInsets={false}
                  contentContainerStyle={
                    Array [
                      Object {
                        "padding": undefined,
                      },
                      undefined,
                    ]
                  }
                  contentInset={
                    Object {
                      "bottom": 0,
                    }
                  }
                  enableAutomaticScroll={true}
                  enableResetScrollToCoords={true}
                  extraHeight={75}
                  extraScrollHeight={0}
                  getScrollResponder={[Function]}
                  handleOnScroll={[Function]}
                  keyboardDismissMode="interactive"
                  keyboardOpeningTime={250}
                  keyboardShouldPersistTaps="handled"
                  keyboardSpace={0}
                  onScroll={[Function]}
                  resetKeyboardSpace={[Function]}
                  resetScrollToCoords={
                    Object {
                      "x": 0,
                      "y": 0,
                    }
                  }
                  scrollEventThrottle={1}
                  scrollForExtraHeightOnAndroid={[Function]}
                  scrollToEnd={[Function]}
                  scrollToFocusedInput={[Function]}
                  scrollToPosition={[Function]}
                  showsVerticalScrollIndicator={true}
                  style={
                    Object {
                      "backgroundColor": "transparent",
                      "flex": 1,
                    }
                  }
                  viewIsInsideTabBar={false}
                  virtual={undefined}
                >
                  <View>
                    <View
                      accessibilityComponentType={undefined}
                      accessibilityLabel={undefined}
                      accessibilityTraits={undefined}
                      accessible={true}
                      collapsable={undefined}
                      hasTVPreferredFocus={undefined}
                      hitSlop={undefined}
                      isTVSelectable={true}
                      nativeID={undefined}
                      onLayout={undefined}
                      onResponderGrant={[Function]}
                      onResponderMove={[Function]}
                      onResponderRelease={[Function]}
                      onResponderTerminate={[Function]}
                      onResponderTerminationRequest={[Function]}
                      onStartShouldSetResponder={[Function]}
                      style={
                        Object {
                          "alignItems": "center",
                          "alignSelf": "center",
                          "backgroundColor": "purple",
                          "borderBottomWidth": null,
                          "borderColor": "#007aff",
                          "borderLeftWidth": null,
                          "borderRadius": 5,
                          "borderRightWidth": null,
                          "borderTopWidth": null,
                          "elevation": 2,
                          "flexDirection": "row",
                          "height": 50,
                          "justifyContent": "center",
                          "marginBottom": 0,
                          "marginTop": 30,
                          "opacity": 1,
                          "paddingBottom": 6,
                          "paddingHorizontal": 50,
                          "paddingTop": 6,
                        }
                      }
                      testID={undefined}
                      tvParallaxProperties={undefined}
                    >
                      <Text
                        accessible={true}
                        allowFontScaling={true}
                        ellipsizeMode="tail"
                        style={
                          Object {
                            "color": "white",
                            "fontSize": 20,
                          }
                        }
                      >
                        Criar
                      </Text>
                    </View>
                  </View>
                </RCTScrollView>
              </View>
            </View>
          </RCTScrollView>
        </View>
      </View>
    </View>
    <View
      collapsable={undefined}
      onLayout={[Function]}
      pointerEvents="box-none"
      style={
        Object {
          "backgroundColor": "#F7F7F7",
          "borderBottomColor": "#A7A7AA",
          "borderBottomWidth": 0.5,
          "height": 64,
          "paddingBottom": 0,
          "paddingLeft": 0,
          "paddingRight": 0,
          "paddingTop": 20,
        }
      }
    >
      <View
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      />
      <View
        style={
          Object {
            "flex": 1,
          }
        }
      >
        <View
          style={
            Object {
              "bottom": 0,
              "flexDirection": "row",
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        >
          <View
            collapsable={undefined}
            pointerEvents="box-none"
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "transparent",
                "bottom": 0,
                "flexDirection": "row",
                "justifyContent": "center",
                "left": 0,
                "opacity": 1,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <Text
              accessibilityTraits="header"
              accessible={true}
              allowFontScaling={true}
              collapsable={undefined}
              ellipsizeMode="tail"
              numberOfLines={1}
              onLayout={[Function]}
              style={
                Object {
                  "color": "rgba(0, 0, 0, .9)",
                  "fontSize": 17,
                  "fontWeight": "700",
                  "marginHorizontal": 16,
                  "textAlign": "center",
                }
              }
            />
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
